modules/KIWILinuxRC.sh

```diff
@@ -1324,8 +1324,10 @@ EOF
     # install grub2 in BIOS mode
     #--------------------------------------
     if [ ! -z "$kiwi_OfwGrub" ];then
+        local prepdev=$imagePrepDevice
+        test -z $prepdev && prepdev=$(ddn $imageDiskDevice $kiwi_OfwGrub)
```

Comment:
I realized that imagePrepDevice is never set by kiwi. This means we either lost something in the past or that code path never worked.

Can you confirm that we need the prep device only for the purpose of installing grub correctly ? if so you can delete the use of $imagePrepDevice entirely and just write

```bash
prepdev=$(ddn $imageDiskDevice $kiwi_OfwGrub)
```